phplib/utils.php

```diff
@@ -238,33 +269,39 @@ static function parse_nagios_host_data($state, $filter, $api_cols) {
                         }
                         $downtime_remaining = null;
                         $downtimes = array_merge($service_detail['downtimes'], $host_detail['downtimes']);
-                        if ($host_detail['scheduled_downtime_depth'] > 0
-                            || $service_detail['scheduled_downtime_depth'] > 0
+                        if ((isset($host_detail['scheduled_downtime_depth']) && $host_detail['scheduled_downtime_depth'] > 0)
+                            || (isset($service_detail['scheduled_downtime_depth']) && $service_detail['scheduled_downtime_depth'] > 0)
```

Comment:
Maybe? I have to look at this bit again. I remember having to add several checks to this part of the code just to get the stack working and probably furiously flung isset()'s everywhere.